### PR TITLE
Create the tagging functionality TE-1926

### PR DESCRIPTION
**Pull request by schenedx**
@rlucioni @cpennington @macdiesel 
Please help review

### CODE REVIEW

File: tubular/github_api.py [hunk 1]
```diff
@@ -276,6 +277,47 @@ def merge_pull_request(self, pr_number):
         pull_request = self.get_pull_request(pr_number)
         pull_request.merge()
 
+    def create_a_tag(
```
Comment (macdiesel) [inline tubular/github_api.py]: Could we rename this to create_tag?

File: tubular/tests/test_github.py [hunk 2]
```diff
@@ -156,6 +156,29 @@ def test_create_pull_request(self, head, base, title, body):
...
+        with patch.object(Github, 'get_user', return_value=mock_user):
+            create_tag_mock = Mock()
+            create_ref_mock = Mock()
+            self.repo_mock.create_git_tag = create_tag_mock
+            self.repo_mock.create_git_ref = create_ref_mock
+
+            test_tag = 'test_tag'
+            test_sha = 'abc'
+            self.api.create_a_tag(test_sha, test_tag)
+            args, kwargs = create_tag_mock.call_args  # pylint: disable=unused-variable, unpacking-non-sequence
```
Comment (macdiesel) [inline tubular/tests/test_github.py]: _, kwargs = ... and I think you won't need the pylint hint.

**Review by macdiesel — APPROVED** [review]
a few small things.

**Comment by schenedx** [comment]
@macdiesel Took in your suggestions.

**Pull request merged**